picker-starter/src/data/categories.ts

```diff
@@ -1,6 +1,3 @@
-import { CategoryItem, ItemQuery } from '@/core'
-import { compareName, delayed, getPage, randomDelay } from '@/utils'
-
 export type MockCategory = {
```

Comment:
Are these mock categories still in use?

Reply:
Yeap, they are used as options for the Combobox filter in the Item tab. 

![Screenshot 2023-12-15 at 11 58 49](https://github.com/storyblok/field-type-examples/assets/1240591/a7bf3f5e-feee-46f6-8e74-e84c03a14d52)